exporter/exporterhelper/common.go

```diff
@@ -27,32 +27,12 @@ type Shutdown func() error
 
 // ExporterOptions contains options concerning how an Exporter is configured.
 type ExporterOptions struct {
-	// TODO: Retry logic must be in the same place as metrics recording because
-	// if a request is retried we should not record metrics otherwise number of
-	// spans received + dropped will be different than the number of received spans
-	// in the receiver.
-	recordMetrics bool
-	recordTrace   bool
```

Comment:
Can you clarify why these are no longer needed?

Reply:
Nevermind: I read the PR description and it says it is now always enabled so options are not needed.